website/docs/operator/data-recovery.md

```diff
@@ -111,10 +110,6 @@ The following is an input event example that an operator might set up while runn
       ]
     }
   },
-  "cumulus_meta": {
-    "system_bucket": "test-internal",
-    "asyncOperationId": "1234"
-  }
```

Comment:
Any reason why this proprty was removed?

Reply:
Reverted this change